lnprototest/runner.py

```diff
@@ -169,6 +173,28 @@ def is_running(self) -> bool:
     def connect(self, event: Event, connprivkey: str) -> None:
         pass
 
+    def send_msg(self, msg: Message) -> None:
+        """Send a message through the last connection"""
+        missing = msg.missing_fields()
+        if missing:
+            raise SpecFileError(self, "Missing fields {}".format(missing))
+        binmsg = io.BytesIO()
+        msg.write(binmsg)
+        self.last_conn.connection.send_message(msg.getvalue())
```

Comment:
why not including the init_msg = runner.recv_msg() inside this function? and return the InitMessage?